react/ui-components/src/atoms/stories/Animation.stories.js

```
@@ -51,26 +51,26 @@ const commonArgs = {
 
-export const SuccessAnimation = Template.bind({});
-SuccessAnimation.args = {
+export const Success = Template.bind({});
+Success.args = {
   ...commonArgs,
   animationData: successAnimation,
 };
 
-export const ErrorAnimation = Template.bind({});
-ErrorAnimation.args = {
+export const Error = Template.bind({});
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)